weblate/templates/snippets/list-objects.html

```diff
@@ -151,6 +151,9 @@
     <td class="actions" rowspan="2">
       <a href="{{ category.get_absolute_url }}" class="btn btn-link green" title="{% trans "Browse" %}" alt="{% trans "Browse" %}">{% icon "folder-multiple-outline.svg" %}</a>
     </td>
+    <td class="actions" rowspan="2">
+      <a href="{{ category.get_absolute_url }}" class="btn btn-link green" title="{% trans "Duplicate" %}" alt="{% trans "Duplicate" %}">{% icon "copy.svg" %}</a>
+    </td>
```

Comment:
I'm not convinced this is a good place for such an action. At least, it should be only shown to users who have permission for that operation.

Moreover, adding another `<td>` will break the layout, it should be included in the existing actions `<td>`.

Reply:
okay, thanks for the comments. Do we need duplicate item button on the `All Components` Page?

Reply:
I think the Manage menu of a component might be a better place.